include/fluent-bit/multiline/flb_ml.h

```diff
@@ -150,6 +150,14 @@ struct flb_ml_parser {
      */
     flb_sds_t key_content;
 
+    /*
+     * The 'key' name that contains the multiline message. For REGEX, ENDSWITH and
```

Comment:
When I will get a POC working, I will take care of comments and makes things like documentation.